examples/mappings/requirements.txt

```diff
@@ -0,0 +1,6 @@
+oteapi-core>=0.6.1,<1
+#oteapi-dlite>0.1.6,<1  # update once after migrating to pydantic v2
+oteapi-dlite @ git+https://github.com/EMMC-ASBL/oteapi-dlite.git@cwa/close-187-migrate-to-pydantic-v2
+DLite-Python>=0.5.1,<1
```

Comment:
This is problematic to include in tests with a different set of requirements. We end up having issues with the dlite-installation when we both install from source and pypi.

Reply:
I see your point. But the purpose of the examples are that you should be able to run the example by just checking it out and install the requirements with pip without needing to compile DLite from source.

Please note that `oteapi-dlite` requires `DLite-Python`, so `DLite-Python` will nevertheless be installed implicitly if we remove it from the requirements file. Better to have it installed explicitly. 

If you want to use the compiled version of DLite, you can do the following:

    pip install -r requirements.txt
    pip uninstall DLite-Python
    cd build-directory
    cmake --install .

Reply:
OK. But remember that this is an example that other people will use to test things out. If you want to keep it like this, at least add an explanation on what can go wrong in the requirements for this example.

Reply:
Added a section to the end of the readme about running the example with DLite installed from source.